app/src/main/java/com/example/nidoqueue/controller/ContextManager.java

```diff
@@ -5,17 +5,19 @@
 import com.example.nidoqueue.activity.AbstractActivity;
 
 public class ContextManager {
-    private static ContextManager contextManager = new ContextManager();
 
-    AbstractActivity context;
+    //private static final ContextManager contextManager = new ContextManager();
 
+    AbstractActivity context;
 
-    private ContextManager(){}
```

Comment:
I think the constructor needs to be private to follow singleton pattern.  If the constructor is private this ensures there is only one instance because no other class can call the constructor. [https://www.tutorialspoint.com/design_pattern/singleton_pattern.htm]

Reply:
See later discussion https://github.com/CMPUT301W21T19/Nidoqueue/pull/67#discussion_r602982705